tests/ngui-e2e/server/README.md

```diff
@@ -1,159 +1,135 @@
-# NGUI E2E Server
+# Lightrail API with Next Gen UI Agent
 
-[![Module Category](https://img.shields.io/badge/Module%20Category-Testing/Evaluation-darkmagenta)](https://github.com/RedHat-UX/next-gen-ui-agent)
-[![Module Status](https://img.shields.io/badge/Module%20Status-Tech%20Preview-orange)](https://github.com/RedHat-UX/next-gen-ui-agent)
+FastAPI server with **Next Gen UI Agent** integration for AI-powered UI component generation using LlamaStack.
 
-This module is part of the [Next Gen UI Agent project](https://github.com/RedHat-UX/next-gen-ui-agent).
+## Features
 
-FastAPI backend server for the NGUI end-to-end testing application. Provides AI-powered UI generation APIs using LangGraph integration with configurable AI models.
+✅ **AI-Powered UI Generation** - Generate rich UI components from natural language  
+✅ **Multiple Component Types** - Cards, tables, videos, images, set-of-cards  
+✅ **Simple Movie Search** - JSON-based movie database with smart search  
+✅ **LlamaStack Integration** - Enterprise LLM with SSL support  
+✅ **CORS-Enabled** - Ready for frontend integration  
 
-## Provides
+## Quick Start
 
-* FastAPI server with NGUI agent integration
-* LangGraph-based AI workflow for UI component generation
-* Configurable AI model support (Ollama, OpenAI, models.corp)
-* RESTful API endpoints for UI generation
-* CORS-enabled for frontend integration
+```bash
+# 1. Install dependencies
+poetry install
 
-## Installation
+# 2. Set environment variables
+export LIGHTRAIL_LLAMA_STACK_BASE_URL="your-llama-stack-url"
```

Comment:
Not sure how much we want to expose LIGTRAIL. Maybe we should use `LLAMA_STACK_BASE_URL` as a default one (for other deployments), and use `LIGHTRAIL_LLAMA_STACK_BASE_URL` as a fallback to be used if the default one is not used?